lms/djangoapps/certificates/views/webview.py

```diff
@@ -113,6 +113,14 @@ def _update_certificate_context(context, course, user_certificate, platform_name
         year=date.year
     )
 
+    # Translators:  EEF format of the date
```

Comment:
@vlasovmichael They say hints for translators should precede the term directly:
```
If you would like to give translators hints about a translatable string, you can add a comment prefixed with the Translators keyword on the line preceding the string
```
(ref https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#comments-for-translators)

Can you please place `# Translators:  EEF format of the date` on the 117th line, right before  the `_('{day}/{month}/{year}'` term?

Reply:
fixed, thanks